README.md

```diff
@@ -31,8 +31,9 @@ By using the expo mobile app, you will be able to test the app in both iOS & And
 - Open the expo
 
 #### View in Android Emulator
-- Download and install [genymotion](https://www.genymotion.com/desktop/)
-- Open the emulator (order is important here)
+- Download and install [genymotion](https://www.genymotion.com/fun-zone/)
+- Genymotion requires a virtualization tool so download [Virtualbox too](https://www.virtualbox.org/wiki/Downloads)
+- Open genymotion (order is important here)
```

Comment:
There's still something wrong with my instructions here.  When I open genymotion, regardless of whether I start an emulator or not, I get errors when I try to "Run on Android device/emulator".

The first error below happened when the emulator (Google Nexus 10 - 4.2.2 - API 17 - 2560x1600) was running, and the second happened after I closed the emulator and tried again.

![image](https://user-images.githubusercontent.com/1809882/46873887-c16b6780-ce05-11e8-833e-c757b6e68f90.png)